pkg/volunteer/volunteer_test.go

```diff
@@ -82,15 +96,25 @@ func TestGenerateRecord(t *testing.T) {
 			},
 			errstr: "fail",
 		},
+		{ // test extensionLister failure, should not cause total failure
+			tweak: func(vol *volunteer) {
```

Comment:
Here we test the case where the `extensionsLister` returns an error and ensure that the overall report generation still succeeds.